pipelines/toast_ground_sim.py

```diff
@@ -1242,22 +1245,55 @@ def output_tidas(args, comm, data, totalname):
     if args.tidas is None:
         return
     autotimer = timing.auto_timer()
-    from toast.tod.tidas import OpTidasExport
+    from toast.tod.tidas import OpTidasExport, TODTidas
```

Comment:
Should we put this import to the top of the file and inside a try-catch clause?

Reply:
I don't think we should import spt3g or tidas unless we need to.  However, thanks for pointing out that this code does not make use of the tidas_available and spt3g_available variables from toast.tod.  Those should be checked (i.e. we already have a try/except check in place- we should just use the result).  I'll make that change shortly.